src/DynamoCore/Nodes/NodeCategories.cs

```diff
@@ -92,6 +92,7 @@ public static class BuiltinNodeCategories
         public const string REVIT_PARAMETERS = "Revit.Parameters";
         public const string REVIT_BAKE = "Revit.Bake";
         public const string REVIT_API = "Revit.API";
+        public const string REVIT_ELEMENTS_DIVIDEDPATH_ACTION = "Revit.Elements.DividedPath.Actions";
```

Comment:
Are you sure about deleting `LOGIC_MATH_ROUNDING`. Even if it's not used in `DynamoCore`, it doesn't mean it's not used somewhere else.
E.g. `DynamoRevit` use some strings from BuiltinNodeCategories, but `DynamoCore` doesn't use it. I'm afraid if I delete something, something from other connected project can be broken.